backend/src/utils/notebookUtils.ts

```diff
@@ -274,7 +274,7 @@ export const assembleNotebook = async (
         'opendatahub.io/username': username,
         'kubeflow-resource-stopped': null,
         'opendatahub.io/accelerator-name':
-          acceleratorProfile.acceleratorProfile?.metadata.name || '',
+          data.acceleratorProfile?.acceleratorProfile?.metadata.name || '',
```

Comment:
The latter `?` is not needed here 🤔 
```suggestion
          data.acceleratorProfile?.acceleratorProfile.metadata.name || '',
```